First commit of startsWith.md (#1637)
Codecademy/docs: content/java/concepts/strings/terms/startsWith/startsWith.md

---
Title: '.startsWith()'
Description: 'Returns true if a string starts with given character(s) and false otherwise.'
Subjects:
  - 'Computer Science'
Tags:
  - 'Booleans'
  - 'Strings'
  - 'Methods'
CatalogContent:
  - 'learn-java'
  - 'paths/computer-science'
---

> The **`.startsWith()`** method returns `true` if a string starts with given character(s). Otherwise, `false` is returned.

Reviewer: ```suggestion
The **`.startsWith()`** method returns `true` if a string starts with a given character sequence. Otherwise, `false` is returned.
```

## Syntax

```pseudo

> string.startsWith(object);

Reviewer: ```suggestion
string.startsWith(stringOfCharacters);
```

```

> The `object` can either be a string literal or a representation of a `String` value. This will return `true` if the `object` has the same starting character sequence as `string` .

Reviewer: This sentence is a bit confusing. Can you reword it? Also, `object` is now `stringOfCharacters`.

## Example

> The following example showcases the `.startsWith()` method:

Reviewer: ```suggestion
The following example demonstrates the `.startsWith()` method:
```

---
Title: '.startsWith()'
Description: 'Returns true if a string starts with a given character sequence and false otherwise.'
Subjects:
  - 'Computer Science'
Tags:
  - 'Booleans'
  - 'Strings'
  - 'Methods'
CatalogContent:
  - 'learn-java'
  - 'paths/computer-science'
---

The **`.startsWith()`** method returns `true` if a string starts with a given character sequence. Otherwise, `false` is returned.

## Syntax

```pseudo
string.startsWith(stringOfCharacters);
```

> The `stringOfChcaracters` can either be a string literal or a representation of a `String` value (Variable/Constant etc). 

Reviewer 2: ```suggestion
The `stringOfChcaracters` can either be a string literal or a representation of a `String` value (variables, constants, etc.). 
```
